courses/free-courses-en.md

```diff
@@ -311,6 +311,7 @@
 
 ### Computer Organization and Architecture
 
+* [Advanced Computer Organization & Architecture (COA)](https://www.youtube.com/playlist?list=PL1iLu2CSC9EWlusZsVqXQ9BFg5jRRLzom) - Smruti R. Sarangi
```

Comment:
```suggestion
* [Advanced Distributed Systems](https://www.youtube.com/playlist?list=PL1iLu2CSC9EWlusZsVqXQ9BFg5jRRLzom) - Smruti R. Sarangi
```
Please add with the same title

Reply:
make this change please

Reply:
Actually looks like they put the wrong link before. @KaranKulshrestha has updated the link to the COA course now. So this can be closed i think